dagql/idtui/golden_test.go

```diff
@@ -159,6 +167,82 @@ func (s TelemetrySuite) TestGolden(ctx context.Context, t *testctx.T) {
 		{Module: "./viztest/typescript", Function: "fail-log", Fail: true},
 		{Module: "./viztest/typescript", Function: "fail-effect", Fail: true},
 		{Module: "./viztest/typescript", Function: "fail-log-native", Fail: true},
+		// local module calls local module fn
+		{
+			Function: "trace-function-calls",
+			DBTest: func(t *testctx.T, db *dagui.DB) {
+				require.NotEmpty(t, db.Spans.Order)
+				var depCalled, rootCalled bool
+				for _, s := range db.Spans.Order {
+					if s.Name == "Dep.getFiles" {
+						require.Equal(t, "TraceFunctionCalls", s.ExtraAttributes[telemetry.ModuleCallerFunctionCallNameAttr])
+						require.Equal(t, "github.com/dagger/dagger/viztest", strings.Split(s.ExtraAttributes[telemetry.ModuleCallerRefAttr].(string), "@")[0])
+						require.Equal(t, "getFiles", s.ExtraAttributes[telemetry.ModuleFunctionCallNameAttr])
+						require.Equal(t, "github.com/dagger/dagger/viztest/dep", strings.Split(s.ExtraAttributes[telemetry.ModuleRefAttr].(string), "@")[0])
+						depCalled = true
+					} else if s.Name == "Viztest.traceFunctionCalls" {
+						require.Equal(t, "", s.ExtraAttributes[telemetry.ModuleCallerFunctionCallNameAttr])
+						require.Contains(t, s.ExtraAttributes[telemetry.ModuleCallerRefAttr], "")
+						require.Equal(t, "traceFunctionCalls", s.ExtraAttributes[telemetry.ModuleFunctionCallNameAttr])
+						require.Contains(t, s.ExtraAttributes[telemetry.ModuleRefAttr], "github.com/dagger/dagger/viztest")
```

Comment:
nit: just missed a spot?

```suggestion
						require.Equal(t, s.ExtraAttributes[telemetry.ModuleCallerRefAttr], "")
						require.Equal(t, "traceFunctionCalls", s.ExtraAttributes[telemetry.ModuleFunctionCallNameAttr])
						require.Equal(t, s.ExtraAttributes[telemetry.ModuleRefAttr], "github.com/dagger/dagger/viztest")
```